src/lib/location/location.ts

```diff
@@ -32,17 +32,17 @@ export const getContinentByCountry = (country: string): string => {
 };
 
 export const getStateIsoByName = (state: string): string => {
-	const stateEntries = Object.entries(states).find(s => s[1] === state);
+	const iso = states[state];
 
-	if (!stateEntries) {
-		throw new Error(`state not found ${state}`);
+	if (!iso) {
+		throw new Error(`iso not found ${state}`);
 	}
 
-	return String(stateEntries[0]);
+	return iso;
 };
 
 export const getStateNameByIso = (iso: string): string => {
-	const state = states[iso as keyof typeof states];
+	const state = _.invert(states)[iso];
```

Comment:
Hmm, so we're using this the other way too. Ok, I guess but should add a test to check that DC resolves to "District Of Columbia" otherwise, we could accidentally change it later just be reordering the lines.